README.md

```diff
@@ -135,12 +136,20 @@ Whether a record can be deleted.
 
 #### `editHeading`
 
-*String* | Default: "Edit"
+*String* | Default: ""
 
 The heading of the edit column in `rh-table` when it is visible.
 
 * * *
 
+#### `deleteHeading`
+
+*String* | Default: ""
```

Comment:
Yeah, the Edit column was documented incorrectly, the default has always been an empty string